src/api/app/controllers/webui/package_controller.rb

```diff
@@ -373,6 +373,11 @@ def branch_diff_info
     }
   end
 
+  def branch_dialog
```

Comment:
We don't have a modal/dialog anymore, so this action name is misleading. I suggest naming this action `branch` and renaming the `branch` action below to `create_branch` (maybe we can find a better name). Then we need to adapt the routes a bit.

We can either have one route `branch` with the `GET` going to `package#branch` and the `POST` going to `package#create_branch`. This is what Rails does for `index` and `create`. They both point to the same route, but depending on the HTTP verb, the action varies.

The other option is to have a separate route, so two routes just like the actions, so `GET branch` and `POST create_branch`.